### PR TITLE
build app binary inside the docker image

### PR DESCRIPTION
**Pull request by Nesze**
Size: 19.18 MB

When you guys approve this I'm going to create the PRs for the rest of the go apps.

### CODE REVIEW

**Comment by catalin-ilea** [comment]
lgtm

**Comment by Zapadlo** [comment]
Can we `ADD` the source and `go build` it instead of go getting it. Its much easier to work locally/on branches this way. Otherwise we always go get master and its a pain if you want to test a branch.

**Comment by Nesze** [comment]
Yes, good idea, so we then use go get only for dependencies and build the project from the added source.

**Comment by Zapadlo** [comment]
Yes, that would be ideal.

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,19 @@
 FROM gliderlabs/alpine:3.2
 
+ADD . /aggregate-healthcheck
+RUN apk --update add go git\
+  && export GOPATH=/.gopath \
+  && go get github.com/Financial-Times/coco-aggregate-healthcheck \
+  && cd aggregate-healthcheck \
+  && go build \
+  && cp aggregate-healthcheck ../coco-aggregate-healthcheck \
```
Comment (Zapadlo) [inline Dockerfile]: `mv` instead please

**Comment by Nesze** [comment]
done

**Comment by Zapadlo** [comment]
+1

**Pull request merged**